pkg/oc/admin/policy/modify_roles.go

```diff
@@ -569,7 +569,7 @@ func (o *RoleModificationOptions) RemoveRole() error {
 		roleBinding.Subjects, cnt = removeSubjects(roleBinding.Subjects, subjectsToRemove)
 		found += cnt
 
-		if len(roleBinding.Subjects) > 0 {
+		if len(roleBinding.Subjects) > 0 || roleBinding.Annotations[ReconcileProtectAnnotation] == "true" {
```

Comment:
We should check for either the RBAC or the authz policy protection annotation.

Reply:
Also does this cover both CRB and RB?

Reply:
conversion gives us the "correct" label, methinks

Reply:
It covers both CRB and RB yeah